CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService jokeService;
+        private IJokeOutput jokeOutput;
```

Comment:
Field should be prefixed with underscore and PascalCase
```suggestion
        private IJokeService _JokeService;
        private IJokeOutput _JokeOutput;
```